styles/components/MyAccount.module.scss

```diff
@@ -96,3 +119,30 @@
   font-weight: 510;
   font-style: normal !important;
 }
+
+.feesNotification {
+  margin-left: 0px !important;
+  background: rgba(250, 244, 244, 1) !important;
+  border-left: 2px solid var(--nypl-colors-ui-error-primary);
+  border-radius: 0px !important;
+  > div > div > div {
+    display: flex;
+    gap: 4px;
+    flex-direction: row;
+    > p {
+      margin-bottom: 0px;
+      color: var(--nypl-colors-ui-error-primary);
+      font-size: var(--nypl-fontSizes-desktop-body-body1);
+    }
+    > svg {
+      fill: var(--nypl-colors-ui-error-primary);
+      margin-top: 3px;
```

Comment:
This might be a question for Apoorva, but do we want to use hardcoded pixel values in these styles or should we use design system spacing/REM values?

Reply:
I would also be in favor of using DS values instead of hard coded custom values. @apoorvakhanolkar would it be okay to use the closest approximation of DS spacing values here instead?

Reply:
noting here that I do think this should be a broader conversation about the My account page designs since the decision on this will affect the styling on other components